notebooks/llm-chatbot/gradio_helper.py

```diff
@@ -110,7 +110,7 @@ def make_demo(run_fn: Callable, stop_fn: Callable, title: str = "OpenVINO Chatbo
                         with gr.Row():
                             top_k = gr.Slider(
                                 label="Top-k",
-                                value=50,
+                                value=1,
```

Comment:
This change affects all models in this notebook, most of them have default value 50. Adding a new model shouldn't affect other models, or we should check this change for all models.

Reply:
greedy search should work for all of models

Reply:
if temperature > 0 , do_sample will set to True , it is multinomial semantically 

What did you get ? Did you get that the output contains garbage with Top k 50 and don't have it with 1 ?

Reply:
“output contains garbage with Top k 50 and don't have it with 1”， yes, exactly.

Reply:
why we need do_sampling in that case ?